test/e2e/page-objects/pages/send/solana-send-page.ts

```diff
@@ -4,23 +4,28 @@ import { Driver } from '../../../webdriver/driver';
 class SendSolanaPage {
   private driver: Driver;
 
-  private readonly sendAmountInput = '#send-amount-input';
+  private readonly clearToAddressField = '#send-clear-button';
 
-  private readonly toAddressInput = '#send-to';
+  private readonly cancelButton = {
+    tag: 'button',
+    testId: 'send-cancel-button-snap-footer-button',
+  };
 
   private readonly continueButton = {
-    text: 'Continue',
     tag: 'button',
```

Comment:
this selector was wrong, as the button element didn't have text, rather there was a nested span with the 'Continue' test. I've added a test id so we can now point to the correct selector each time